core/web/schema/type/eth_key.graphql

```diff
@@ -0,0 +1,11 @@
+type EthKey {
```

Comment:
Missing `EthBalance` and `LinkBalance`

Reply:
Probably unrelated to this PR but it would be nice to read the balance from the BalanceMonitor if possible. Then we can avoid extra queries to the eth node and return a lot faster